docs/ADRs/0014-written-language.md

```diff
@@ -0,0 +1,55 @@
+# ADR-0014 Written Language
+
+## Context
+
+In order to expand our reach and foster international collaboration in the field of AI Validation, we have decided to
+conduct all communication in English on public platforms such as GitHub. This decision aims to facilitate better
+understanding and participation from our global colleagues. However, within the Dutch Government, the norm is to
```

Comment:
Official translation of Rijksoverheid in English is Government of the Netherlands. 

```suggestion
understanding and participation from our global colleagues. However, within the Government of the Netherlands, the norm is to
```